src/frontend/planner_test/tests/testdata/output/temporal_filter.yaml

```diff
@@ -72,18 +72,28 @@
         └──  StreamExchange Broadcast from 2
 
     Fragment 1
-    StreamProject { exprs: [(AtTimeZone((AtTimeZone(now, 'UTC':Varchar) - '00:00:00':Interval), 'UTC':Varchar) - '02:00:00':Interval) as $expr1], output_watermarks: [$expr1] }
+    StreamProject { exprs: [SubtractWithTimeZone(now, '02:00:00':Interval, 'UTC':Varchar) as $expr1], output_watermarks: [$expr1] }
     └──  StreamNow { output: [now] } { state table: 5 }
 
     Fragment 2
-    StreamProject { exprs: [(AtTimeZone((AtTimeZone(now, 'UTC':Varchar) - '00:00:00':Interval), 'UTC':Varchar) - '01:00:00':Interval) as $expr2], output_watermarks: [$expr2] }
+    StreamProject { exprs: [SubtractWithTimeZone(now, '01:00:00':Interval, 'UTC':Varchar) as $expr2], output_watermarks: [$expr2] }
     └──  StreamNow { output: [now] } { state table: 6 }
 
-    Table 0 { columns: [ t1_ts, t1__row_id ], primary key: [ $0 ASC, $1 ASC ], value indices: [ 0, 1 ], distribution key: [ 1 ], read pk prefix len hint: 1 }
+    Table 0
+    ├── columns: [ t1_ts, t1__row_id ]
+    ├── primary key: [ $0 ASC, $1 ASC ]
+    ├── value indices: [ 0, 1 ]
+    ├── distribution key: [ 1 ]
+    └── read pk prefix len hint: 1
```

Comment:
Could you please take a look? @ice1000 I think the content does not changes or get larger. :thinking:

Reply:
This is probably because the other lines get shorter, so the pretty printer now realize that this thing can fit in a shorter width.

Before this PR, the other lines are already longer than this line, so it didn't make sense to crop down the struct.

Reply:
To keep the old behavior, set a very large initial width in `PrettyConfig`

Reply:
Thanks for the explanation. I am not going to tune the width in config here, as it brings more unrelated format changes.

This behavior makes sense in an interactive environment. But for snapshot testing, we may not want to reformat some lines based on widths of other lines 😂  cc @xxchan

Reply:
> the other lines are already longer than this line, so it didn't make sense to crop down the struct.

@ice1000  I'm not sure how reasonable is it. How is the "max width" dynamically adjusted? (become larger and larger?) And from my understanding, the initial width in the config represents the "preference", so maybe we don't want to update it when printing multiple things. 🤔

Reply:
What if the preferred width is 80, and there is a single field whose content is a string, and has length 999? In this case, if you have another struct, which is like 120 in total but can crop down to fit into 80, do you still crop it down or put it in a single line? Currently the library put it in a single line

Reply:
The behavior is explained in the rfc draft. I admit it's a bit involved. If you'd like to suggest a change, it should go to the rfc first (the pretty printer is a fully faithful implementation of the rfc)